addons/website_forum/views/forum.xml

```diff
@@ -245,7 +245,8 @@
                         </group>
                         <group name="answers" string="Answers" attrs="{'invisible':[('parent_id','!=',False)]}">
                             <field name="child_ids" nolabel="1">
-                                <tree>
+                                <tree decoration-muted="active == False">
```

Comment:
```suggestion
                                <tree decoration-muted="not active" context="{'active_test': False}">
```
This should be the only diff (with the active field added below).
If it doesnt work, it is a bug/regression from previous version (you can see there is multiple occurence in the code that should work.

https://github.com/odoo/odoo/blame/023e2e636c97f07b360fab649c36a178e0c16793/addons/mail/views/mail_channel_views.xml#L82
https://github.com/odoo/odoo/blame/023e2e636c97f07b360fab649c36a178e0c16793/odoo/addons/base/views/ir_ui_view_views.xml#L39

Reply:
AFAIK 'active_test': False doesn't works on tree tag

In this case it is also not Working on field, I have checked the example you have given and it has no effect of active_test

Reply:
Indeed it is not working anymore, as I mentioned this is then a regression.
We will check after the Odoo v15 freeze why is this not working, because it should/was.

Once we found why it is not working (anymore), we will be able to move forward here, as the ideal diff here would just be to add that active_test context key on the o2m field inside the form view.
(It was working at some point when working on it at https://github.com/odoo/odoo/pull/63040/commits/83714b2ffc71c1b58fdeb69f55e8039c6654af40)